code/processes/dqc.q

```diff
@@ -276,12 +278,9 @@ reruncheck:{[chkid]
   /- clear .u.end timer
   .timer.removefunc'[exec funcparam from .timer.timer where `.u.end in' funcparam];
   delete configtable from `.dqe;
-  .dqe.currentpartition:pt+1;
-  /- Checking whether .eodtime.nextroll is correct as it affects periodic writedown
-  if[(`timestamp$.dqe.currentpartition)>=.eodtime.nextroll;
-    .eodtime.nextroll:.eodtime.getroll[`timestamp$.dqe.currentpartition];
-    .lg.o[`dqc;"Moving .eodtime.nextroll to match current partition"]
-    ];
+  .dqe.currentpartition:(`date^.dqe.partitiontype)$(.z.D,.z.d).dqe.gmttime;
```

Comment:
Not sure if I approved this before or it just slipped me by but this `gmttime` variable seems a little awkward to me. Is this kind of thing used elsewhere in TorQ? Can than the manual aspect of needing to update the variable be removed using `.proc.cp`? Also wondering if it should be referencing UTC rather than GMT.

Reply:
The gmttime variable is in wdb, also a boolean. Will change it to refer to UTC.